node/errors.js

```diff
@@ -445,34 +444,71 @@ module.exports.classify = function classify(err) {
         case 'tchannel.max-pending-for-service':
             return 'Declined';
 
-        case 'tchannel.timeout':
+        case 'tchannel.socket-local-closed':
+        case 'tchannel.local.reset':
+            return 'Cancelled';
```

Comment:
Whether local close or reset corresponds to a deliberate cancellation is debatable.

I do suspect that these cases should be candidates for retry.

These cases do not indicate remote health problems, so like cancellation and bad request, they should not trip circuit breaking. Perhaps we need another error class.

Whether deliberate and accidental cancellation should have the same retry semantics is the question.